include/bbp/sonata/report_reader.h

```diff
@@ -18,9 +18,10 @@ namespace sonata {
 // KeyType will be NodeID for somas report and pair<NodeID, uint32_t> for elements report
 template <typename KeyType>
 struct SONATA_API DataFrame {
-    using DataType = std::map<KeyType, std::vector<float>>;
-    std::vector<double> index;
-    DataType data;
+    using DataType = std::vector<KeyType>;
+    std::vector<double> times;
+    DataType ids;
+    std::vector<std::vector<float>> data;
```

Comment:
once we are sure about the first and second indices for data we can add a comment here:
// data[ids][times] 
or 
// data[times][ids]